courses/migrations/0025_nullify_expiration_date.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.10 on 2020-02-27 14:37
+
+from django.db import migrations, models
+
+
+def nullify_courserun_expiration_dates(apps, schema_editor):
+    """
+    Fetch all Courses/Programs without a readable_id and set that field to
+    a slug-ified version of the object's title
+    """
+    CourseRun = apps.get_model("courses", "CourseRun")
+    CourseRun.objects.update(expiration_date=None)
```

Comment:
reference to your statement: 

> Make sure all courses with the empty expiration date have links on the dashboard to edX course. Only courses that have explicitly set expiration date in the past should not show the link.

So what does it mean, all the expiration date will be `none` for all actual data and the link will appear over the dashboard for expired runs too ??

Reply:
This migration is to remove the expiration dates already set previously, as was required by the marketing team. Going forward the expiration date won't be populated automatically and will need to be set manually when needed.